cmd/bridge/main.go

```diff
@@ -60,6 +60,7 @@ func main() {
 	fK8sMode := fs.String("k8s-mode", "in-cluster", "in-cluster | off-cluster")
 	fK8sModeOffClusterEndpoint := fs.String("k8s-mode-off-cluster-endpoint", "", "URL of the Kubernetes API server.")
 	fK8sModeOffClusterSkipVerifyTLS := fs.Bool("k8s-mode-off-cluster-skip-verify-tls", false, "DEV ONLY. When true, skip verification of certs presented by k8s API server.")
+	fK8sModeOffClusterCAFile := fs.String("k8s-mode-off-cluster-ca-file", "", "PEM file for Kubernetes API server, will be igrored if k8s-mode-off-cluster-skip-verify-tls is true, and use ca-file if not given")
```

Comment:
This wording could be smoothed out just a bit more.  `and use ca-file if not given` is a bit terse & could be confusing.

Reply:
Yeah, seems it is confusing